docs/source/developers/continuous_integration/crossbow.rst

```diff
@@ -115,8 +115,6 @@ to step 3:
    username and ``ghrepo`` is the GitHub repository name (typically
    ``crossbow``):
 
-   - **CircleCI**: ``https://circleci.com/gh/<ghuser>/<ghrepo>/edit#env-vars``
-
    - **TravisCI**: ``https://travis-ci.com/<ghuser>/<ghrepo>/settings``
 
      - Confirm the `auto cancellation`_ feature is turned off for branch builds. This should be the default.
```

Comment:
Remove Appveyor item below and replace it with a Github Actions item?

Reply:
Just to clarify, it looks like there are a few references to AppVeyor in `dev/archery/archery/crossbow/core.py`. Is AppVeyor no longer in use at all for Crossbow?

Reply:
It's not, but the old code to generate AppVeyor configuration and launch builds is probably still there just in case.

Reply:
Thanks for confirming! I'll go ahead an remove all mentions of AppVeyor from the Crossbow documentation.

Reply:
@pitrou, when I set up a queue repository, I did not need to further configure the repository to use the GitHub Personal Access Token that was created in step 4. Do you think there is any additional setup or configuration that we should call out for GitHub Actions in step 6?